### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by Tinkaa**
Adding simple (not prettified) scripts to compute mock-up data for food insecurity indicators for Ethiopia and Somalia. Adding the output of this data to the dashboard/data folder. Also moved the shapefiles to that folder.

### CODE REVIEW

File: analyses/somalia/scripts/som_foodinsec_trigger.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+df=pd.concat([df_fn,df_gipc])
+df=df.replace(0,np.nan)
+df["date"]=pd.to_datetime(df["date"])
+df["year"]=df["date"].dt.year
+df["month"]=df["date"].dt.month
+df["trigger_ML1_4_20"]=df.apply(lambda x: define_trigger_percentage(x,"ML1",4,20),axis=1)
+df["trigger_ML1_3_30"]=df.apply(lambda x: define_trigger_percentage(x,"ML1",3,30),axis=1)
+df["trigger_ML1_3_5i"]=df.apply(lambda x: define_trigger_increase(x,"ML1",3,5),axis=1)
+df[f"threshold_reached_ML1"]=np.where((df[f"trigger_ML1_4_20"]==1) | ((df[f"trigger_ML1_3_30"]==1) & (df[f"trigger_ML1_3_5i"]==1)),1,0)
```
Comment (joseepoirier) [inline analyses/somalia/scripts/som_foodinsec_trigger.py:48]: the trigger is:
The projected national population in Phase 3 and above exceed 20%, AND
(The national population in Phase 3 is projected to increase by 5 percentage points, OR
The projected national population in Phase 4 or above is 2.5%)

File: analyses/somalia/scripts/som_foodinsec_trigger.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+df["trigger_ML2_4_20"]=df.apply(lambda x: define_trigger_percentage(x,"ML2",4,20),axis=1)
+df["trigger_ML2_3_30"]=df.apply(lambda x: define_trigger_percentage(x,"ML2",3,30),axis=1)
+df["trigger_ML2_3_5i"]=df.apply(lambda x: define_trigger_increase(x,"ML2",3,5),axis=1)
+df[f"threshold_reached_ML2"]=np.where((df[f"trigger_ML2_4_20"]==1) | ((df[f"trigger_ML2_3_30"]==1) & (df[f"trigger_ML2_3_5i"]==1)),1,0)
```
Comment (joseepoirier) [inline analyses/somalia/scripts/som_foodinsec_trigger.py:53]: The projected national population in Phase 3 and above exceed 20%, AND
(The national population in Phase 3 is projected to increase by 5 percentage points, OR
The projected national population in Phase 4 or above is 2.5%)

File: analyses/somalia/scripts/som_foodinsec_trigger.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+df["trigger_ML1_3_20"] = df.apply(lambda x: define_trigger_percentage(x,"ML1",3,20),axis=1)
+df["trigger_ML1_3_5in"] = df.apply(lambda x: define_trigger_increase(x,"ML1",3,5),axis=1)
+df["trigger_ML1_4_2half"] = df.apply(lambda x: define_trigger_percentage(x,"ML1",4,2.5),axis=1)
+
+df["trigger_ML2_3_20"] = df.apply(lambda x: define_trigger_percentage(x,"ML2",3,20),axis=1)
+df["trigger_ML2_3_5in"] = df.apply(lambda x: define_trigger_increase(x,"ML2",3,5),axis=1)
+df["trigger_ML2_4_2half"] = df.apply(lambda x: define_trigger_percentage(x,"ML2",4,2.5),axis=1)
+
+# determine whether national trigger is met
+df['threshold_reached_ML1'] =  np.where((df['trigger_ML1_3_20']==1) & ((df['trigger_ML1_3_5in'] )==1 | (df['trigger_ML1_4_2half'] == 1)), 1, 0)
```
Comment (joseepoirier) [inline analyses/somalia/scripts/som_foodinsec_trigger.py:64]: The projected national population in Phase 3 and above exceed 20%, AND
(The national population in Phase 3 is projected to increase by 5 percentage points, OR
The projected national population in Phase 4 or above is 2.5%)

File: analyses/somalia/scripts/som_foodinsec_trigger.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+df['threshold_reached_ML2'] =  np.where((df['trigger_ML2_3_20']==1) & ((df['trigger_ML2_3_5in'] )==1 | (df['trigger_ML2_4_2half'] == 1)), 1, 0)
```
Comment (joseepoirier) [inline analyses/somalia/scripts/som_foodinsec_trigger.py:65]: The projected national population in Phase 3 and above exceed 20%, AND
(The national population in Phase 3 is projected to increase by 5 percentage points, OR
The projected national population in Phase 4 or above is 2.5%)

**Review by joseepoirier — COMMENTED** [review]
I think the logic of the trigger is wrong. The modifications to the R file are ok.

**Comment by Tinkaa** [comment]
Oh shoot, forgot to remove the old example code. Thanks loads for being so alert, should be fixed now.

**Pull request merged**